### ISSUE TITLE
nix build

### ISSUE THREAD

**Issue by RyanGibb** [lazamar/nix-package-versions#40]
I got the project building with `docker build --tag lazamar/nix-package-versions:latest .` from the GitHub action, but a Nix build would be nice for a Nix project.

**Comment by lazamar** [comment]
Indeed, but I only ever build this to re-create the database and so far Docker has been enough.
#39 adds a flakes dev shell, but I haven't tried it. 
Would that work for what you are trying to do?
